sopel_modules/rat_board.py

```diff
@@ -1707,8 +1713,11 @@ def func_delete(bot, trigger, id):
             return
         bot.say('Deleted case with id ' + str(id) + ' - THIS IS NOT REVERTIBLE!')
     else:
-        result = callapi(bot, 'GET', uri='/rescues?data={"markedForDeletion":{"marked":true}}',
-                         triggernick=str(trigger.nick))
+        try:
+            result = callapi(bot, 'GET', uri='/rescues?data={"markedForDeletion":{"marked":true}}', triggernick=str(trigger.nick))
+        except ratlib.api.http.APIError:
+            bot.reply('Couldn\'t find cases marked for deletion due to API error.')
```

Comment:
do **not** silently suppress exceptions. please do at very least log them somewhere.
```suggestion
            logging.exception("failed to find MFD cases due to API error.)
            bot.reply('Couldn\'t find cases marked for deletion due to API error.')
```

Reply:
Very little exception catching is logged in mecha 2. Instead we opt to reply to the executor that there was an error. In fact in some cases we have logging disabled (commented out)

Reply:
in various cases this codebase has commented out `print` statements, instead of making logging calls. print != logging.